src/components/meeting/Meeting.js

```diff
@@ -3,23 +3,32 @@ import {useParams} from "react-router-dom";
 import {interviewAPI} from "../../utils/apis/InterviewAPI";
 import {MeetingDisplayName} from "./MeetingDisplayName";
 import {JitsiWrapper} from "./JitsiComponentWrapper";
+import {CircularProgress} from "@material-ui/core";
 
 export const Meeting = () => {
     let { interviewId, companyId } = useParams()
     const [name, setName] = useState(null)
+    const [fetching, setFetching] = useState(true)
 
     useEffect(() => {
         if(companyId === undefined) {
             interviewAPI.getJobSeekerNameByInterviewId(interviewId)
-                .then(data => setName(data))
+                .then(data => {
+                    setName(data?.name)
+                    setFetching(false)
+                })
+        } else {
+            setFetching(false)
         }
     }, [companyId, interviewId])
 
     return (
         <div>
-            {name === null ?
+            {
+                fetching ? <CircularProgress /> :
+                name === null ?
                 <MeetingDisplayName roomName={`${interviewId}`}/>
-                : <JitsiWrapper admin={false} roomName={`${interviewId}`} displayName={name}/>
+                : <JitsiWrapper admin={false} roomName={`${interviewId}`} displayName={name ? name : "John Doe"}/>
```

Comment:
O co chodzi z tym John Doe?

Reply:
No na razie nie mam jak pobrać imienia typa (interview service i jego zasadność jest do przegadania), A jakieś imie dla robaka musi być, do tej pory było Interviewee 1 chyba. John Doe to jest chyba określenie na wszystkie osoby w kostnicy, których nikt nie zidentyfikował, więc idealnie pasuje XDDD

Reply:
A tu przed wejściem do rozmowy nie miało być możliwości wpisania swojej nazwy w pokoju?

Reply:
Nie, nazwę, jak się nazywasz i tylko dla devów
Dev z robakiem by się raczej nie dogadali na nazwę pokoju.

Reply:
Znaczy chodziło mi o to twoją nazwę jak będziesz się wyświetlał w pokoju

Reply:
No, to to jest